cmd/istanbul/cmd.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/rlp"
+	atypes "github.com/getamis/go-ethereum/core/types"
+	"github.com/naoina/toml"
+	"github.com/urfave/cli"
+)
+
+var (
+	decodeCommand = cli.Command{
+		Action:    decode,
+		Name:      "decode",
+		Usage:     "To decode a Istanbul extraData",
+		ArgsUsage: "<extra data>",
+		Flags: []cli.Flag{
+			ExtraDataFlag,
+		},
+		Description: `
+The extraData command will decode extraData for the given input which should be a hex string.
+`,
+	}
+
+	encodeCommand = cli.Command{
+		Action:    encode,
+		Name:      "encode",
+		Usage:     "To encode a Istanbul extraData",
+		ArgsUsage: "<config file>",
+		Flags: []cli.Flag{
+			ConfigFlag,
+		},
+		Description: `
+The extraData command will encode Istanbul extraData for the given input file.
```

Comment:
This command encodes vanity and validators to extraData. Please refer to example/config.toml.